core/org.openjdk.jmc.common/src/main/java/org/openjdk/jmc/common/security/PersistentCredentials.java

```diff
@@ -30,10 +30,13 @@
  * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY
  * WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package org.openjdk.jmc.ui.common.security;
+package org.openjdk.jmc.common.security;
 
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import org.openjdk.jmc.common.security.ISecurityManager;
+import org.openjdk.jmc.common.security.SecurityException;
+import org.openjdk.jmc.common.security.SecurityManagerFactory;
```

Comment:
Are you sure these imports are needed? The imported classes were migrated as well...

Reply:
Thanks for catching these, yes they're not needed I'll remove them.